src/icons/imported/Icon/Content/FileBookmark32.js

```diff
@@ -1,10 +1,10 @@
 // @flow
```

Comment:
Can you make folder path lowercase.
Also, can "imported" be dropped now from the path, since we have new top level folders now